EDA_analysis.Rmd

```diff
@@ -0,0 +1,80 @@
+---
+title: "Exoloratory Data Analysis of the Covid-19 Data Set"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message=FALSE)
+library(tidyverse)
+library(knitr)
+library(broom)
+library(datateachr)
+library(infer)
+library(cowplot)
+theme_set(theme_minimal())
+set.seed(2020)
+```
+
+## Summary of the data set
+
+The data set used in this project is a collection of the COVID-19 data maintained by [Our World in Data](https://ourworldindata.org/coronavirus). It is updated daily and includes data on confirmed cases, deaths, hospitalizations and testing, as well as other variable of potential interests. This data has been collected, aggregated, and documented by Cameron Appel et al. Various origins have been sourced. Confirmed cases, deaths, hospitalizations and other related information are sourced from the European Centre for Disease Prevention and Control (ECDC). Testing related information are collected by the Our World in Data team from offical reports. Other sources include United Nations, World Bank and Global Burden of Disease. More information can be found [here](https://github.com/owid/covid-19-data/blob/master/public/data/owid-covid-codebook.csv).
```

Comment:
Can you please add [@owidcoronavirus] after Cameron Appel et al., since that's a reference? Thanks